alignak/action.py

```diff
@@ -136,7 +136,6 @@ class ActionBase(AlignakObject):
     def __init__(self, params=None, parsing=True):
         super(ActionBase, self).__init__(params, parsing=parsing)
         self.creation_time = time.time()
```

Comment:
I think if have params not define creation_time, exist_status, fill_default and log_actions, because it's filled with serialize data